sklearn/feature_selection/from_model.py

```diff
@@ -169,6 +169,12 @@ def fit(self, X, y=None, **fit_params):
         self.estimator_.fit(X, y, **fit_params)
         return self
 
+    @property
+    def threshold_(self):
+        scores = _get_feature_importances(self.estimator_, self.norm_order)
```

Comment:
If we cache scores, then the property might still require recomputing the mean or std. But we can't avoid that if we want ``threshold_`` to be up-to-date with ``threshold`` unless we overwrite ``setattr`` which I'm not sure is a good idea ;)